Add StatsFormat to the config.json file

vdemeester opened the pull request:
As for `ps`, `images`, `network ls` and `volume ls`, this makes it possible to define a custom default format.

```
{
    "HttpHeaders": {
      "MyHeader": "MyValue"
    },
    "psFormat": "table {{.ID}}\\t{{.Image}}\\t{{.Command}}\\t{{.Labels}}",
    "imagesFormat": "table {{.ID}}\\t{{.Repository}}\\t{{.Tag}}\\t{{.CreatedAt}}",
    "statsFormat": "table {{.Container}}\\t{{.CPUPerc}}\\t{{.MemUsage}}",
    "detachKeys": "ctrl-e,e"
}
```

/cc @thaJeztah @dnephin 

🐸

Signed-off-by: Vincent Demeester vincent@sbr.pm

thaJeztah commented:
design 👍

thaJeztah commented:
can you update the docs as well? https://github.com/docker/docker/blob/master/docs/reference/commandline/cli.md#configuration-files

LGTM otherwise

thaJeztah commented:
perhaps needs a test :smile:

vdemeester committed:
Add StatsFormat to the config.json file

As for `ps`, `images`, `network ls` and `volume ls`, this makes it
possible to define a custom default format.

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

ripcurld00d commented:
LGTM :+1:

thaJeztah reviewed (approved):
LGTM

cpuguy83 commented:
LGTM

cpuguy83 commented:
Moved to docs

thaJeztah commented:
docs LGTM as well
